src/util/weather_api_util.js

```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery'
+
+export const fetchForecast = (cityName) => {
+  const API_KEY = process.env.REACT_APP_WEATHER_API_KEY;
+  return $.ajax({
+    method: 'GET',
+    url:  `https://api.openweathermap.org/data/2.5/forecast?q=${cityName},us&appid=${API_KEY}`
+  })
+};
+
+export const fetchCurrentWeather = (cityName) => {
+  const API_KEY = process.env.REACT_APP_WEATHER_API_KEY;
+  return $.ajax({
+    method: 'GET',
+    url:  `https://api.openweathermap.org/data/2.5/weather?q=${cityName},us&appid=${API_KEY}`
+  })
+};
```

Comment:
Curious why you chose jquery and ajax here? What are some other options?

Reply:
I chose Jquery and Ajax because I was most familiar with it, especially when using it with the full redux lifecycle build.. but some other options could be to use axios or even just a 'fetch' call with a promise returned inside the component itself.  I implemented that here: 

<img width="908" alt="Screen Shot 2020-11-11 at 10 39 45 PM" src="https://user-images.githubusercontent.com/19714255/98892698-dbf1ee00-246e-11eb-898b-d511d902b553.png">

Reply:
Nice! I recommend running away from using jQuery (or even AJAX) in a React app, or in general at this point. But, of course, that's just my two cents!